### PR TITLE
Use newer versions

### PR DESCRIPTION
**Pull request by Jokeswar**

### CODE REVIEW

File: nomad_jobs/acs-interface.hcl [hunk 1]
```diff
@@ -61,7 +63,7 @@ job "acs-interface" {
       }
       driver = "docker"
       config {
-        image = "vmck/acs-interface:interface"
+        image = "vmck/acs-interface"
```
Comment (mgax) [inline nomad_jobs/acs-interface.hcl]: Let's tag versions so we know exactly what is running. Can you please tag a version already?

File: nomad_jobs/acs-interface.hcl [hunk 2]
```diff
@@ -146,4 +149,4 @@ job "acs-interface" {
       }
     }
   }
-}
```
Comment (mgax) [inline nomad_jobs/acs-interface.hcl]: Whoops :)

**Review by mgax — CHANGES_REQUESTED** [review]
:+1: but let's tag everything.

File: main.tf [hunk 3]
```diff
@@ -18,3 +18,7 @@ resource "nomad_job" "vmck" {
 resource "nomad_job" "acs-interface" {
   jobspec = "${file("${path.module}/nomad_jobs/acs-interface.hcl")}"
 }
+
+resource "nomad_job" "ingress" {
+  jobspec = "${file("${path.module}/nomad_jobs/ingress.hcl")}"
+}
```
Comment (mgax) [inline main.tf]: Hmm why is this bit showing up? Can you please merge with master? (`git fetch; git merge origin/master; git push`)

**Review by mgax — CHANGES_REQUESTED** [review]

File: main.tf (hunk 3 above)
Comment (Jokeswar) [reply main.tf]: Didn't help

Comment (mgax) [reply main.tf]: I think you merged something else - `Merge branch 'update-hcl' of github.com:vmck/acs-deploy into update-hcl` - I was suggesting `origin/master`.

**Review by mgax — APPROVED** [review]
Cool! Please apply and merge.

**Pull request merged**